From 0b4fc045ddd875ffa845a2dbd2bff0d22d617e46 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Mon, 22 Apr 2013 18:29:53 +0300
Subject: [PORT FROM MAIN] Telephony: SPN name not initialized.

BZ: 11860

Old spn record cleared upon instantiation and disposal
of SIMRecords.

Change-Id: I614479d05d92f8017e7039d6feecdf4a79f8bfcb
Orig-Change-Id: Ib64542234457abf098820d3b3b0fb9ef0eec9bed
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 .../internal/telephony/uicc/SIMRecords.java        |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/java/com/android/internal/telephony/uicc/SIMRecords.java b/src/java/com/android/internal/telephony/uicc/SIMRecords.java
index 2927e86..6b6c7f2 100644
--- a/src/java/com/android/internal/telephony/uicc/SIMRecords.java
+++ b/src/java/com/android/internal/telephony/uicc/SIMRecords.java
@@ -222,6 +222,7 @@ public class SIMRecords extends IccRecords {
         mIccId = null;
         // -1 means no EF_SPN found; treat accordingly.
         mSpnDisplayCondition = -1;
+        mSpn = null;
         mEfMWIS = null;
         mEfCPHS_MWI = null;
         mSpdiNetworks = null;
-- 
1.7.4.1

